tests/tests/tier0/bluechictl-set-get-default-target/main.fmf

```diff
@@ -0,0 +1,2 @@
+summary: Test the set and get default target function
```

Comment:
I'd rephrase to `Test bluechictl get-default and set-default commands`

Reply:
Changed